jorvik/storage/isolation.py

```diff
@@ -0,0 +1,58 @@
+from typing import Callable
+
+from pyspark.sql import DataFrame, SparkSession
+from pyspark.sql.streaming import StreamingQuery
+
+from jorvik.storage import Storage
+from jorvik.storage.basic import BasicStorage
+
+class IsolatedStorage:
+    def __init__(self, storage: Storage, verbose: bool = False, isolation_provider: Callable = None):
+        self.storage = storage
+        self.verbose = verbose
+        self.isolation_provider = isolation_provider
+
+    def _values_to_list(self, value: str) -> list:
+        """ Normalize any user input into a clean list of values. """
+        if isinstance(value, str):
+            return [item.strip() for item in value.split(",") if item.strip()]
+        else:
+            raise ValueError("Expected input to be a single or comma separated string")
```

Comment:
The fact that it needs to be a string is communicated by the type annotation, maybe the Exception is redundant?